src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="form form--big"
+      action="#"
+      method="get"
+      data-qa="keypress"
+    >
+      <input
+        class="search search--big"
+        type="search"
+        name="bigSearch"
+        data-qa="big"
+        placeholder="Try &#34;Los Angeles&#34;"
+      >
+      <button class="search__btn search__btn--big">
+        <img
+          class="search__img--big"
+          src="./images/Search.svg"
+          alt="Search big"
+        >
+      </button>
```

Comment:
Why this is a button? It needs to be a part of input as background or simple image, not a button